{
  "Header": {
    "Original": "CVE-2021-31566 (An improper link resolution flaw can occur while extracting an archive ...)",
    "Line": 64292,
    "ID": "CVE-2021-31566",
    "Description": "(An improper link resolution flaw can occur while extracting an archive ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2987-1}",
      "Line": 64293,
      "Type": "xref",
      "Bugs": [
        "DLA-2987-1"
      ]
    },
    {
      "Original": "- libarchive 3.5.2-1 (bug #1001990)",
      "Line": 64294,
      "Type": "package",
      "Package": "libarchive",
      "Kind": "fixed",
      "Version": "3.5.2-1",
      "BugNo": 1001990
    },
    {
      "Original": "[bullseye] - libarchive 3.4.3-2+deb11u1",
      "Line": 64295,
      "Type": "package",
      "Release": "bullseye",
      "Package": "libarchive",
      "Kind": "fixed",
      "Version": "3.4.3-2+deb11u1"
    },
    {
      "Original": "[buster] - libarchive \u003cno-dsa\u003e (Minor issue)",
      "Line": 64296,
      "Type": "package",
      "Release": "buster",
      "Package": "libarchive",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/libarchive/libarchive/issues/1566",
      "Line": 64297,
      "Type": "NOTE",
      "Description": "https://github.com/libarchive/libarchive/issues/1566"
    },
    {
      "Original": "NOTE: https://github.com/libarchive/libarchive/commit/b41daecb5ccb4c8e3b2c53fd6147109fc12c3043 (v3.5.2)",
      "Line": 64298,
      "Type": "NOTE",
      "Description": "https://github.com/libarchive/libarchive/commit/b41daecb5ccb4c8e3b2c53fd6147109fc12c3043 (v3.5.2)"
    },
    {
      "Original": "NOTE: https://github.com/libarchive/libarchive/commit/e2ad1a2c3064fa9eba6274b3641c4c1beed25c0b (v3.5.2)",
      "Line": 64299,
      "Type": "NOTE",
      "Description": "https://github.com/libarchive/libarchive/commit/e2ad1a2c3064fa9eba6274b3641c4c1beed25c0b (v3.5.2)"
    }
  ]
}